--- DBM-Deep-2.0008/lib/DBM/Deep.pod.orig	2012-06-17 21:58:48.000000000 +0200
+++ DBM-Deep-2.0008/lib/DBM/Deep.pod	2013-06-22 11:01:30.783679073 +0200
@@ -51,7 +51,7 @@
 
 B<NOTE>: 2.0000 introduces Unicode support in the File back end. This
 necessitates a change in the file format. The version 1.0003 format is
-still supported, though, so we have added a L</db_version|db_version()>
+still supported, though, so we have added a L<db_version|db_version()>
 method. If you are using a database in the old format, you will have to
 upgrade it to get Unicode support.
 
